adapter.js

```diff
@@ -268,7 +268,11 @@ export default function ({ config, asyncFetch, headers, handleResponse }) {
       )
       .bichain(
         handleRejectedResponse,
-        (res) => Async.Resolved({ ok: true, results: res.items }),
+        (res) =>
+          Async.Resolved({
+            ok: true,
+            results: map((o) => ({ ok: true, id: o.index._id }), res.items),
```

Comment:
is `_id` the internal id used by elasticsearch? Should we expose that, or should we instead return the index name in the `id` field?

Reply:
My thoughts is because `name` is consumer provided, and what the consumer will use to reference and consume the index on subsequent calls.

Reply:
According to my testing with open search and the code in the adapter the `_id` field is the 'id' field and the results from opensearch only return the `_id`

adapter bulk reduce

```
reduce(
            (
              arr,
              doc,
            ) => [...arr, { index: { _index: index, _id: doc.id } }, doc],
            [],
          ),
     ```

result.items

```
[
    {
      "index": {
        "_index": "app-movies-ckutqvg7y002q07l7h2qd5gmthyper63default",
        "_type": "_doc",
        "_id": "1",
        "_version": 2,
        "result": "updated",
        "_shards": {
          "total": 2,
          "successful": 2,
          "failed": 0
        },
        "_seq_no": 1,
        "_primary_term": 1,
        "status": 200
      }
    },
    {
      "index": {
        "_index": "app-movies-ckutqvg7y002q07l7h2qd5gmthyper63default",
        "_type": "_doc",
        "_id": "2",
        "_version": 2,
        "result": "updated",
        "_shards": {
          "total": 2,
          "successful": 2,
          "failed": 0
        },
        "_seq_no": 1,
        "_primary_term": 1,
        "status": 200
      }
    }
    ```
    
  happy to return `name` or `id` but they do not appear in the result set of a bulk submission, do you know of flags that are set, to return `id` or `name` also, by the looks of the reduce `_id` is the same as `id`. 
  
  thanks  tom

Reply:
Oh okay I see, so `index._id`is the `_id` provided for the document being indexed. Makes sense to return it here. Thanks for that clarification.